express-notes-api/index.js

```diff
@@ -0,0 +1,90 @@
+const express = require('express');
+const app = express();
+const data = require('./data.json');
+const fs = require('fs');
+
+app.use(express.json());
+
+const newJSON = JSON.stringify(data, null, 2);
+
+app.get('/api/notes', (req, res) => {
+  const array = [];
+  if(!data.notes){
+    return [];
```

Comment:
you don't want to return an empty array, you want to send it to the front end. You can delete lines 12 to 14 since if there are no notes and the loop runs, you will still send an empty array to the client